processor/k8sattributesprocessor/metadata.yaml

```diff
@@ -8,6 +8,10 @@ status:
 
 # resource attributes are exposed through a different configuration interface (extract::metadata).
 resource_attributes:
+  k8s.cluster.uid:
+    description: Gives cluster uid identified with kube-system namespace
+    type: string
+    enabled: true
```

Comment:
This attribute should be added as optional (disabled by default), otherwise it's a breaking change

Reply:
Okay, thanks for reviewing.
Are there any other changes needed apart from this? Would add all the changes.
+ I need to add some tests.

Reply:
Done